### PR TITLE
Updated documentation for public class/variables and some formatting

### PR DESCRIPTION
**Pull request by ts-leojoseph-r**
# Description
* Added some documentation for public APIs
* Refactoring

# Checklist
- [x] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `fastlane ci` without errors

### CODE REVIEW

File: MiniApp/Classes/Extensions/MiniApp+FileManager.swift [hunk 1]
```diff
@@ -35,8 +35,14 @@ extension FileManager {
     class func getMiniAppVersionDirectory(usingAppId id: String) -> URL? {
         do {
             let miniAppDirectory = getMiniAppDirectory(with: id)
-            let versionDirectory = try FileManager.default.contentsOfDirectory(at: miniAppDirectory, includingPropertiesForKeys: [URLResourceKey.isDirectoryKey], options: [.skipsHiddenFiles])[0]
-            return versionDirectory
+            var isDir: ObjCBool = true
+            if FileManager.default.fileExists(
+                atPath: miniAppDirectory.path,
+                isDirectory: &isDir) {
```
Comment (corycaywood) [inline MiniApp/Classes/Extensions/MiniApp+FileManager.swift:41]: Why does this need to be passed as a variable instead of just directly passing `true`?

**Review by corycaywood — APPROVED** [review]
Looks good! 👍

File: MiniApp/Classes/Extensions/MiniApp+FileManager.swift (hunk 1 above)
Comment (ts-leojoseph-r) [reply MiniApp/Classes/Extensions/MiniApp+FileManager.swift:41]: We need to pass a [UnsafeMutablePointer](https://developer.apple.com/documentation/foundation/filemanager/1410277-fileexists) - ObjC boolean here. I wish I would know a better way to do here, but sort of followed the old way.

Comment (Climbatize) [reply MiniApp/Classes/Extensions/MiniApp+FileManager.swift:41]: yeah, unfortunately this is a C++ legacy from ObjC.

**Review by Climbatize — APPROVED** [review]
LGTM

**Pull request merged**